<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[Title is not matching: ]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="29381" started-at="2017-11-03T12:56:05Z" finished-at="2017-11-03T12:56:34Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="29381" started-at="2017-11-03T12:56:05Z" finished-at="2017-11-03T12:56:34Z">
      <class name="com.caratlane.test.VerifySortingTest">
        <test-method status="PASS" signature="preCondition()[pri:0, instance:com.caratlane.test.VerifySortingTest@ff5b51f]" name="preCondition" is-config="true" duration-ms="6118" started-at="2017-11-03T18:26:05Z" finished-at="2017-11-03T18:26:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preCondition -->
        <test-method status="FAIL" signature="testVerifyingSort()[pri:0, instance:com.caratlane.test.VerifySortingTest@ff5b51f]" name="testVerifyingSort" duration-ms="22709" started-at="2017-11-03T18:26:11Z" finished-at="2017-11-03T18:26:34Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.fail(Assert.java:100)
	at com.caratlane.generic.BasePage.verifyTitle(BasePage.java:28)
	at com.caratlane.page.HomePage.verifyingTitle(HomePage.java:22)
	at com.caratlane.test.VerifySortingTest.testVerifyingSort(VerifySortingTest.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Title is not matching: ]]>
            </line>
          </reporter-output>
        </test-method> <!-- testVerifyingSort -->
        <test-method status="PASS" signature="postCondition()[pri:0, instance:com.caratlane.test.VerifySortingTest@ff5b51f]" name="postCondition" is-config="true" duration-ms="164" started-at="2017-11-03T18:26:34Z" finished-at="2017-11-03T18:26:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postCondition -->
      </class> <!-- com.caratlane.test.VerifySortingTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
